src/Reva.ts

```diff
@@ -42,13 +46,16 @@ export interface RevaValidateOptions {
   options?: Partial<RevaOptions>;
 }
 
+export const revaDefaultParamAjvOptions = {
+  allErrors: true,
+  coerceTypes: true,
+  strict: false,
+};
+export const revaDefaultBodyAjvOptions = { allErrors: true, strict: false };
+
 export class Reva {
-  private paramAjv = new Ajv({
-    allErrors: true,
-    coerceTypes: true,
-    strict: false,
-  });
-  private bodyAjv = new Ajv({ allErrors: true, strict: false });
+  private paramAjv = new Ajv(revaDefaultParamAjvOptions);
```

Comment:
Let's change this line to `private bodyAjv: Ajv` so we don't have to call `new Ajv` twice. (same for line 58)

Reply:
Thanks for checking & feedback.
Updated.